[2024-02-18T00:39:44.272+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Read_and_load_of_API_to_MySQLDB.companies_table scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-18T00:39:44.293+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Read_and_load_of_API_to_MySQLDB.companies_table scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-18T00:39:44.294+0000] {taskinstance.py:2170} INFO - Starting attempt 7 of 12
[2024-02-18T00:39:44.326+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): companies_table> on 2024-02-11 00:00:00+00:00
[2024-02-18T00:39:44.336+0000] {standard_task_runner.py:60} INFO - Started process 347 to run task
[2024-02-18T00:39:44.341+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Read_and_load_of_API_to_MySQLDB', 'companies_table', 'scheduled__2024-02-11T00:00:00+00:00', '--job-id', '495', '--raw', '--subdir', 'DAGS_FOLDER/CRM.py', '--cfg-path', '/tmp/tmpyhjqnbxd']
[2024-02-18T00:39:44.347+0000] {standard_task_runner.py:88} INFO - Job 495: Subtask companies_table
[2024-02-18T00:39:44.566+0000] {task_command.py:423} INFO - Running <TaskInstance: Read_and_load_of_API_to_MySQLDB.companies_table scheduled__2024-02-11T00:00:00+00:00 [running]> on host 091c4b0f3ebe
[2024-02-18T00:39:45.082+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jmoc' AIRFLOW_CTX_DAG_ID='Read_and_load_of_API_to_MySQLDB' AIRFLOW_CTX_TASK_ID='companies_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-11T00:00:00+00:00'
[2024-02-18T00:39:45.146+0000] {base.py:83} INFO - Using connection ID 'mysql_connection' for task execution.
[2024-02-18T00:39:45.232+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS ***.companies(
              id INT AUTO_INCREMENT  PRIMARY KEY NOT NULL,
              name VARCHAR(25) NOT NULL UNIQUE  ,
              catchPhrase VARCHAR(225) NOT NULL,
              bs VARCHAR(50) NOT NULL 
            );
            , parameters: None
[2024-02-18T00:39:45.272+0000] {sql.py:459} INFO - Rows affected: 0
[2024-02-18T00:39:45.295+0000] {base.py:83} INFO - Using connection ID 'postgres_connection' for task execution.
[2024-02-18T00:39:45.315+0000] {sql.py:450} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS companies(
              id BIGSERIAL PRIMARY KEY NOT NULL,
              name VARCHAR(25) NOT NULL UNIQUE  ,
              catchPhrase VARCHAR(225) NOT NULL,
              bs VARCHAR(50) NOT NULL 
            );
            , parameters: None
[2024-02-18T00:39:45.405+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-18T00:39:45.436+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Read_and_load_of_API_to_MySQLDB, task_id=companies_table, execution_date=20240211T000000, start_date=20240218T003944, end_date=20240218T003945
[2024-02-18T00:39:45.533+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-18T00:39:45.596+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
